radio/src/lua/api_general.cpp

```diff
@@ -553,6 +553,26 @@ static int luaGetRotEncSpeed(lua_State * L)
   return 1;
 }
 
+/*luadoc
+@function getRotEncInvert()
+
+Return rotary encoder inverted status
+
+@retval number in list: OFF = 0, ON = 1, V-N = 2, V-A = 3
+  return 0 on radio without rotary encoder
+
+@status current Introduced in 2.8.0
+*/
+static int luaGetRotEncInvert(lua_State * L)
+{
+#if defined(ROTARY_ENCODER_NAVIGATION)
+  lua_pushunsigned(L, g_eeGeneral.rotEncDirection);
+#else
+  lua_pushunsigned(L, 0);
+#endif
+  return 1;
+}
+
```

Comment:
is "...Invert..." the right name? Maybe "getRotEncMode" would be better.

Reply:
Since this now describes more than rotary encoder inversion, yes... it's now getRotEncMode